automation/taskcluster/decision_task.py

```diff
@@ -27,7 +27,7 @@
 
 
 def create_task(name, description, command, scopes = []):
-    return create_raw_task(name, description, "./gradlew --no-daemon clean %s" % command, scopes)
+    return create_raw_task(name, description, "./gradlew --no-daemon clean && ./gradlew --no-daemon %s" % command, scopes)
```

Comment:
What's the reason for running gradle twice? Isn't there some startup penalty since we are not using the daemon?

Reply:
I remember that older versions of gradle had an issue where `clean` wouldn't provide the same "fresh-slate" guarantees if `gradle` was run in parallel, maybe that problem still exists today?

Reply:
@mitchhentges @pocmo I think that's what I'm seeing, yeah. I've captured my observations in the commit that made this change. Copy/pasting here:

> On the new worker type, the old pattern of running 'clean' as part of the
module task is interacting poorly with parallel builds, and we're getting
IO errors. This change ensures we wait until clean is done before doing
any other work.
> Locally this makes no difference (on Grisha's machine, at least) - and the
old pattern worked perfectly fine with the shared worker type.